kfdefs/base/jupyterhub/notebook-images/ope.yaml

```diff
@@ -0,0 +1,30 @@
+---
+kind: ImageStream
+apiVersion: image.openshift.io/v1
+metadata:
+  name: ucsls
```

Comment:
New imagestream must not have the same name as existing imagestream 
https://github.com/operate-first/apps/blob/cfb9f5a1288bcb10fe533f1c1003856b27364365/kfdefs/base/jupyterhub/notebook-images/ucsls.yaml#L5

please change this to **ope**, so that it matches the file name.

Reply:
oops thanks.